### ISSUE TITLE
ImplicitlyUnwrappedOptional types cannot be found in Swift 4.1

### ISSUE THREAD

**Issue by vchmezov** [ivlevAstef/DITranquillity#116]
Hi!

My app fails to validate container under swift 4.1 due to the presence of ImplicitlyUnwrappedOptional properties. DITranquility just cannot find corresponding types.

I found following lines at Helpers.swift:

```
#if swift(>=4.1)
#else
extension ImplicitlyUnwrappedOptional: SwiftTypeGetter {
  static var type: DIAType { return Wrapped.self }
}
#endif
```

When I remove compilation conditions it works as it did under swift 4.0

Was this conditions set by mistake?

**Comment by ivlevAstef** [comment]
Hi!

Yes @Nekitosss fix this bug soon, and I will release today evening.

But is worth already now to start thinking about - https://github.com/apple/swift-evolution/blob/master/proposals/0054-abolish-iuo.md

**Comment by ivlevAstef** [comment]
Fixed in version 3.2.3
